scout/constants/variant_tags.py

```diff
@@ -96,6 +96,15 @@
     ("XD_dn", "X Linked Dominant De Novo"),
 )
 
+# Used for filter variants in blueprints.variants
+VARIANT_GENOTYPES = (
+    ("", ""),
+    ("0/1", "Heterozygous (0/1)"),
```

Comment:
Heterozygous could also be "1/0". 😊 There would be several options to fix; e.g. having a list to match in, or a single pattern for matching that allows both..

Reply:
I think I wrote it like this because the vast majority of calls in our database are of type 0/1 (84%)

<img width="1094" alt="image" src="https://user-images.githubusercontent.com/28093618/200311369-5be09b59-d156-4ced-a4a4-4772efafc05c.png">

Only 2% is the other heterozygous type (1/0)

<img width="1103" alt="image" src="https://user-images.githubusercontent.com/28093618/200311570-f0705546-dcd3-4f55-8c36-107cfa5c44d7.png">

So this 2% gets collected when you filter for "other".

But I agree with you: it's confusing and if you want heterozygous calls then you want them for both alleles. Will fix!